Content.Shared/Movement/Pulling/Systems/PullingSystem.cs

```diff
@@ -152,38 +152,28 @@ private void OnAfterState(Entity<PullerComponent> ent, ref AfterAutoHandleStateE
 
     private void OnDropHandItems(EntityUid uid, PullerComponent pullerComp, DropHandItemsEvent args)
     {
-        if (pullerComp.Pulling == null || pullerComp.NeedsHands)
+        if (pullerComp.NeedsHands)
             return;
 
-        if (!TryComp(pullerComp.Pulling, out PullableComponent? pullableComp))
-            return;
-
-        TryStopPull(pullerComp.Pulling.Value, pullableComp, uid);
+        TryStopPull(pullerComp.Pulling, uid);
     }
 
     private void OnStopPullingAlert(Entity<PullerComponent> ent, ref StopPullingAlertEvent args)
     {
         if (args.Handled)
             return;
-        if (!TryComp<PullableComponent>(ent.Comp.Pulling, out var pullable))
-            return;
-        args.Handled = TryStopPull(ent.Comp.Pulling.Value, pullable, ent);
+
+        args.Handled = TryStopPull(ent.Comp.Pulling, ent);
     }
 
     private void OnPullerContainerInsert(Entity<PullerComponent> ent, ref EntGotInsertedIntoContainerMessage args)
     {
-        if (ent.Comp.Pulling == null)
-            return;
-
-        if (!TryComp(ent.Comp.Pulling.Value, out PullableComponent? pulling))
-            return;
-
-        TryStopPull(ent.Comp.Pulling.Value, pulling, ent.Owner);
```

Comment:
TryStopPull will get his PullableComponent no need to it verbose, null testing is also not needed